cmd/service_env_set.go

```diff
@@ -42,7 +42,7 @@ At least one environment variable must be specified via the --env flag. Specify
 }
 
 func init() {
-	serviceEnvSetCmd.Flags().StringSliceVarP(&flagServiceEnvSetEnvVars, "env", "e", []string{}, "Environment variables to set [e.g. KEY=value]")
+	serviceEnvSetCmd.Flags().StringArrayVarP(&flagServiceEnvSetEnvVars, "env", "e", []string{}, "Environment variables to set [e.g. KEY=value]")
```

Comment:
OK, so this just eliminates the comma-separated-value functionality entirely; you have to specify the flag multiple times to set multiple vars. 👍

Reply:
Ya, the comma-separated functionality wasn't documented in help, so I thought it was ok to remove it :)